lbcapi/api.py

```diff
@@ -44,9 +51,14 @@ def call(self, method, url, params={}, stream=False, files=None):
         if url.startswith(self.server):
             url = url[len(self.server):]
 
-        # Convert parameters into list of tuples.
-        # This makes request encoding to keep the order
-        params = params.items()
+        if sys.version_info <= (3, 0):
+            # If not py3 - convert parameters into list of tuples.
+            # This makes request encoding to keep the order.
+            # In py3 "requests" lib fails with list of tuples,
+            # so the dict is necessary.
```

Comment:
I think there can never be enough testing, as things inside requests might change, and one of the most common support request from developers we get is related to invalid signature. Because of those support requests, I would like to give this library 101 % valid.

But no worries.. A quick check revealed that it is possible to construct GET request in advance and get the parameters, the same that is done with POST requests.

I've been a little busy now, but I'll handle this PR in some point in future when I have time.